src/Compilers/CSharp/Portable/BoundTree/Statement.cs

```diff
@@ -146,20 +146,50 @@ partial class BoundForEachStatement : IForEachLoopStatement
 
     partial class BoundSwitchStatement : ISwitchStatement
     {
+        private static readonly ConditionalWeakTable<BoundSwitchStatement, object> s_switchSectionsMappings =
```

Comment:
Because `ConditionalWeakTable` requires both type arguments to be reference type.